packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -224,8 +231,10 @@ export class PayPalClientError extends Error {
     }
     this.raw = raw;
     this.data = data;
+    this.errorCode = data.L?.length ? parseInt(data.L[0].ERRORCODE) : undefined;
```

Comment:
Currently we only pull the first error, we may need to change this later to error sets depending on how common it is that PayPal uses those.